src/test/resources/edu/cornell/kfs/krad/service/impl/AccountCustomAddressTest.xml

```diff
@@ -97,9 +97,6 @@
     <budgetRecordingLevelCode>O</budgetRecordingLevelCode>
     <accountSufficientFundsCode>N</accountSufficientFundsCode>
     <pendingAcctSufficientFundsIndicator>false</pendingAcctSufficientFundsIndicator>
-    <extrnlFinEncumSufficntFndIndicator>false</extrnlFinEncumSufficntFndIndicator>
-    <intrnlFinEncumSufficntFndIndicator>false</intrnlFinEncumSufficntFndIndicator>
-    <finPreencumSufficientFundIndicator>false</finPreencumSufficientFundIndicator>

```

Comment:
For at least some of the test files involving these 3 removed fields, I would suggest keeping the fields in place on the `<oldData>` side, to help verify that the XML upgrades rules are auto-removing them properly.

Reply:
updated